### DISCUSSION TITLE
is it possible to rowItem has changable height

### DISCUSSION THREAD

**Discussion by OrhanOzkercin** [TanStack/virtual#623] [Q&A]
I want to use a virtualized list but there is one case for me. I have a component that is like an accordion. It can switch on and off which means its height can change. 

I wonder is possible to handle this situation with the library.

**Comment by piecyk** [comment]
Yes, with dynamic case, checkout the https://tanstack.com/virtual/v3/docs/examples/react/dynamic

**Comment by OrhanOzkercin** [comment]
Hello @piecyk, thank you very much for the response. 

I wonder if is it possible to make a row changeable, I could not figure out how dynamic case handles this. 
In my case let's say my row heights are not the same like 200px, 220px, etc., and also when I click a random rowItem that certain item heights update to 500px I want that item to push others below.
I also want to do it with an infinite scroll.

**Reply by piecyk** [reply]
This should be handled automatically when you change row height, under the hood we use resize observer to watch of size changes.
